frontends/web/src/routes/buy/info.tsx

```diff
@@ -110,7 +110,7 @@ class BuyInfo extends Component<Props, State> {
         if (options === undefined) {
             return <Spinner text={t('loading')} />;
         }
-        const name = (code && isBitcoin(code)) ? 'Bitcoin' : 'crypto';
+        const name = (code && isBitcoin(code)) ? 'Bitcoin' : t('buy.info.crypto');
```

Comment:
@benma this is used as `{{name}}` in the translation, should I rename to `{{cryptoOrBitcoin}}` to make it more clear WDYT?

Reply:
keep, save to effort to change it in all languages